StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/RoundedTextField.swift

```diff
@@ -675,37 +685,70 @@ struct RoundedTextField_Previews: PreviewProvider {
                 RoundedTextFieldUIViewRepresentable(
                     placeholder: "Routing number",
                     footerText: nil,
-                    errorText: nil
+                    errorText: nil,
+                    isFocused: false,
+                    theme: .light
                 )
                 .frame(height: 56)
                 RoundedTextFieldUIViewRepresentable(
                     placeholder: "Account number",
                     footerText: "Your account can be checkings or savings.",
-                    errorText: nil
+                    errorText: nil,
+                    isFocused: false,
+                    theme: .light
                 )
                 .frame(height: 80)
                 RoundedTextFieldUIViewRepresentable(
                     placeholder: "Confirm account number",
                     footerText: nil,
-                    errorText: nil
+                    errorText: nil,
+                    isFocused: false,
+                    theme: .light
                 )
                 .frame(height: 56)
                 RoundedTextFieldUIViewRepresentable(
                     placeholder: "Routing number",
                     footerText: nil,
-                    errorText: "Routing number is required."
+                    errorText: "Routing number is required.",
+                    isFocused: false,
+                    theme: .light
                 )
                 .frame(height: 80)
                 RoundedTextFieldUIViewRepresentable(
                     placeholder: "Account number",
                     footerText: "Your account can be checkings or savings.",
-                    errorText: "Account number is required."
+                    errorText: "Account number is required.",
+                    isFocused: false,
+                    theme: .light
                 )
                 .frame(height: 80)
                 Spacer()
             }
             .padding()
             .background(Color(UIColor.customBackgroundColor))
+
+            // Use separate devices to showcase highlighted state
+            RoundedTextFieldUIViewRepresentable(
+                placeholder: "Light theme",
+                footerText: nil,
+                errorText: nil,
+                isFocused: true,
+                theme: .light
+            )
+            .frame(height: 56)
+            .padding()
+            .previewDisplayName("Focused - Light theme")
+
+            RoundedTextFieldUIViewRepresentable(
+                placeholder: "Link Light theme",
+                footerText: nil,
+                errorText: nil,
+                isFocused: true,
+                theme: .linkLight
+            )
+            .frame(height: 56)
+            .padding()
+            .previewDisplayName("Focused - Link Light theme")
```

Comment:
In order to showcase the different focused state themes, we had to use separate SwiftUI preview devices:

https://github.com/user-attachments/assets/83c6ad99-69d3-4b56-aa21-c0498ab7427f